tests/failpoints/cases/test_snap.rs

```diff
@@ -243,6 +243,51 @@ fn test_destroy_peer_on_pending_snapshot() {
     must_get_equal(&cluster.get_engine(3), b"k120", b"v1");
 }
 
+#[test]
+fn test_destroy_peer_on_abort_snapshot() {
+    let mut cluster = new_server_cluster(0, 3);
+    cluster.cfg.raft_store.raft_base_tick_interval = ReadableDuration::millis(10);
+    cluster.cfg.raft_store.raft_election_timeout_ticks = 25; // > lease 240ms
+    cluster.cfg.raft_store.hibernate_regions = false;
+    configure_for_snapshot(&mut cluster);
+    let pd_client = Arc::clone(&cluster.pd_client);
+    pd_client.disable_default_operator();
+
+    let r1 = cluster.run_conf_change();
+    pd_client.must_add_peer(r1, new_peer(2, 2));
+    pd_client.must_add_peer(r1, new_peer(3, 1003));
+
+    cluster.must_put(b"k1", b"v1");
+    // Ensure peer 3 is initialized.
+    must_get_equal(&cluster.get_engine(3), b"k1", b"v1");
+
+    cluster.must_transfer_leader(1, new_peer(1, 1));
+
+    cluster.add_send_filter(IsolationFilterFactory::new(3));
+
+    for i in 0..20 {
+        cluster.must_put(format!("k1{}", i).as_bytes(), b"v1");
+    }
+
+    let apply_snapshot_fp = "region_apply_snap_abort";
+    fail::cfg(apply_snapshot_fp, "return()").unwrap();
+
+    cluster.clear_send_filters();
+    // Wait for leader send snapshot.
+    sleep_ms(100);
+
+    cluster.add_send_filter(IsolationFilterFactory::new(3));
+
+    for i in 0..40 {
```

Comment:
This doesn't seem necessary.

Reply:
Good point. I thought It's unnecessary too. But the result is that without it, the case will pass without fix---self.fsm.peer.raft_group.tick() is called, but it just does not panic. In fact if I use "for i in 0..20" the case won't break either without fix.